src/compas_rhino/artists/robotmodelartist.py

```diff
@@ -109,28 +109,37 @@ def _exit_layer(self):
         self.redraw()
 
     def draw_collision(self):
+        """Draws all collision geometry of the robot model. Return list of newly created Rhino object's GUID."""
```

Comment:
This isn't quite in the standard format for docstrings. A more consistent version would be ```
"""Draw all the collision geometries of the robot model.

Returns
--------
list 
    The GUIDs of the created Rhino objects.
    
"""```  The same applies to `draw_visual`.